index.js

```diff
@@ -1,14 +1,17 @@
-export default function mathClamp(number, {minimum, maximum}) {
-	if (minimum > maximum) {
-		throw new RangeError('`minimum` should be lower than `maximum`');
+export default function mathClamp(number, {min, minimum, max, maximum}) {
+	min ??= minimum;
```

Comment:
Worth adding a comment about it being for back-compat, and for later removal?

Reply:
Yes